modules/authentication/proc-creating-a-custom-transformer-to-provision-users-from-rhsso-to-the-software-catalog.adoc

```diff
@@ -0,0 +1,87 @@
+[id="provisioning-users-from-rhsso-to-the-software-catalog"]
+= Creating a custom transformer to provision users from Red Hat Single-Sign On (RHSSO) to the software catalog
```

Comment:
Have we created attributes for Red Hat Single Sign-On, similar to what we have for OSD and OCP?

Reply:
No

Reply:
I think we need an attribute

Reply:
Not under the scope of this task.